tests/bench/run-benchmarks.ts

```
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
+/**
+ * Script to run benchmarks and generate reports
+ */
+import { execSync } from 'child_process';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)